11ty/definitions/antisemitism.md

```diff
@@ -21,17 +23,25 @@ Antisemitism can arise in many different ways, including:
 -  religious, which includes discrimination against Jewish religious beliefs and religious teachings by other groups implying the inferiority of Jews;
 -  and political, which implies Jews seek national and/or world power.
 
-Antisemitic ideology includes dehuminisation and degradation of Jews, as well as myths which adapt over time.
+Antisemitic ideology includes dehuminisation and degradation of Jews.
+
+## Origin in brief
+
+Antisemitism began in European Christianity as ideological oppression, and functions to protect the ruling class and divert blame for hardship. While it began as a religious intolerance, antisemitism has always in-part been xenophobic.
 
 ## How it works
 
-Many groups are oppressed due to a fixed hierarchy, such as anti-Black racism in the United States. Historically, antisemitism has been described as cyclical. Jewish communities rotate between periods of stability, including some success, and periods of increased negative visibility which sometimes lead to violence. This further perpetuates myths about Jewish power.
+Many groups are oppressed due to a fixed hierarchy, such as anti-Black racism in the United States. Historically, antisemitism has been described as cyclical. Jewish communities rotate between periods of stability, including some success, and periods of increased negative visibility which sometimes lead to violence. This cycle further perpetuates myths about Jewish power.
 
-Antisemitism tends to be most visible when people are forced to compete for scarce resources.
+Antisemitism tends to be most visible in moments when people are forced to compete for scarce resources.
 
-## Origin
+## Jewish diversity
 
-Antisemitism began in European Christianity as ideological oppression, and functions to protect the ruling class and divert blame for hardship. While it began as a religious intolerance, antisemitism has always in-part been xenophobic.
+It's important to understand that Jews are a diverse group of people, inclusive of various ethnicities and races. 
+
+Some Jews come from European and Russian ancestry (Ashkenazi); Middle Eastern, North African, Central Asian, and Balkan (Mizrahi); Ethiopian and Ugandan (African); and Spanish and Portuguese (Sephardi).
+
+There are Jews who are white, Black, Hispanic, Asian, Indigenous, and every census classification of race. There are mixed-race Jews whose ancestors could include any combination. Jews are a global, multi-racial, multi-ethnic people.
 
```

Comment:
Something that I've written but have not included:

> There are people who have converted to Judaism, or people who have parents or other ancestors who have converted to Judaism. 
> 
> ["Who is a Jew?"](https://globaljews.org/resources/research/counting-jews) is a complicated question which requires a decision about who is counted. This can be distinguished by biology, self-identity, or behavior. Biology is the most complex as it is defined differently by the different [religious movements](https://en.wikipedia.org/wiki/Jewish_religious_movements). Self-identity counts people who refer to themselves as Jewish, regardless of their parentage or standing in any particular movement. Behavior includes those who engage in Jewish practices (e.g., going to synagogue, keeping kosher, and other Jewish practices).

Reply:
I agree that this might be a topic best suited for an entry focused on defining Judaism/Jews, instead of here! I want to make sure that this entry covers enough context to define its term, but not tread trying to define multiple terms within it, if that makes sense.

How do you feel about us saving some of what you've written and cross-linking Judaism/Jews to this entry later? At that point, we can also make any necessary adjustments to this term.

Reply:
Sounds good to me. This comment will live on in the PR, so I can certainly resurface it for an entry on Jews/Judaism in the future, and can then revisit what's in the antisemitism article.

Any other changes to make in the meantime?